src/plugins/pdfPlayer/plugin.js

```diff
@@ -278,16 +278,21 @@ export class PdfPlayer {
 
     renderPage(canvas, number) {
         this.book.getPage(number).then(page => {
-            const original = page.getViewport({ scale: 1 });
+            const width = dom.getWindowSize().innerWidth;
+            const height = dom.getWindowSize().innerHeight;
+            const viewport = page.getViewport({ scale: 5 });
```

Comment:
Should the scale be the pixel density of the screen?

Reply:
Thank you for your comment.
I am not sure. My phone's screen has more pixel density than my monitor. 
By scaling high and resizing the pdf depending of the screen size, fixed the problem. I put 5 but 3 was enough for my device.